### ISSUE TITLE
Clarify `startc/src` if working from `start`

### ISSUE THREAD

**Issue by Kubik42** [OpenLiberty/guides-common#90]

**Comment by evelinec** [comment]
@JunyiSun @Kubik42 can we close this one?

**Comment by yeekangc** [comment]
More to this issue ...

Per @Kubik42:
_I talked about this with Jay and Laura a while back when we discussed removing the cd start from the "Getting started" section. Their concern was that if the users skip the "Try what you'll build" section, then they might be lost as to where to begin. So, Jay suggested to put cd start in the first section of the guide that talks about implementation details._

We should settle on 1 approach and adopt it across all guides.

**Comment by evelinec** [comment]
I'm not sure I like this kind of command, `cd ../start` in the guide. We'll discuss further on a common approach and use it for all guides.

**Comment by evelinec** [comment]
Based on the "More to this issue" comment, looks like this is one approach: 

- Get rid of "Now, navigate back to the start directory to begin." under "Try what you'll build" section.
- Move the sentence with this context "navigate to the `start` directory" to the section where user starts working with the implementation.

**Comment by yeekangc** [comment]
Yes, we should avoid `cd ../start` here unless we are certain that it won't cause issues or confusion.

**Comment by JunyiSun** [comment]
@evelinec @yeekangc What's the final decision on this issue?

**Comment by evelinec** [comment]
I already posted my option in my last comment.
